examples/ivr_outbound_dialer.py

```diff
@@ -0,0 +1,374 @@
+#Switchy IVR Outbound Dialer
+#
+#Author: Nenad Corbic <ncorbic@sangoma.com>
+#
+#This application is designed to demonstrate Switchy capabilities
```

Comment:
You should try to keep the comment lines white space consistent. You do it correctly further down.
From a pep8 perspective what you do later is correct: https://www.python.org/dev/peps/pep-0008/#block-comments